lib/install/validate/validator.go

```diff
@@ -156,6 +157,30 @@ func NewValidator(ctx context.Context, input *data.Data) (*Validator, error) {
 	return v, nil
 }
 
+var schemeMatch = regexp.MustCompile(`^\w+://`)
+
+// Starting from Go 1.8 the URL parser does not
```

Comment:
This might be misleading comment. Is a Go bug? If so, then a reference to that bug needs to be added.
If it is a corrected behavior that was not good before then change a comment to be something like:
"ParseURL parses URL adding default https scheme if no scheme is provided."